lusid-sdk-python/tests/test_instrument_master.py

```diff
@@ -0,0 +1,230 @@
+import unittest
+import requests
+import json
+import uuid
+import os
+from datetime import datetime
+import pytz
+import lusid
+import lusid.models as models
+from unittest import TestCase
+from collections import namedtuple
+from msrest.authentication import BasicTokenAuthentication
+from time import sleep
+
+try:
+    # Python 3.x
+    from urllib.request import pathname2url
+except ImportError:
+    # Python 2.7
+    from urllib import pathname2url
+
+class TestFinbourneApi(TestCase):
+    client = None
+    instrumentIds = []
+
+
+    @classmethod
+    def setUpClass(cls):
+
+        cls.ISIN_PROPERTY_KEY = "Instrument/default/Isin"
+        cls.SEDOL_PROPERTY_KEY = "Instrument/default/Sedol"
+        cls.FIGI_SCHEME = "Figi"
+        cls.CUSTOM_INTERNAL_SCHEME = "ClientInternal"
+
+        #   load configuration
+        dir_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
could you refactor this similar to test_finbourne_client.py so that it checks for environment variables first

Reply:
Done - I updated the entire method to reflect Mike's new setup code